File: MsgCommunicator/DatabasePack/Demos/MySQLDAC/C++ Builder 2006/Client/uStart.cpp
//---------------------------------------------------------------------------
#include <vcl.h>
#pragma hdrstop

#include "uStart.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma resource "*.dfm"
TfmStart *fmStart;
//---------------------------------------------------------------------------
__fastcall TfmStart::TfmStart(TComponent* Owner)
  : TForm(Owner)
{
}
//---------------------------------------------------------------------------
void __fastcall TfmStart::FormShow(TObject *Sender)
{
 ModalResult = mrOk;
 bnOK->SetFocus();  
}
//---------------------------------------------------------------------------

File: MsgCommunicator/Demos/_Databases/MySQLDAC/Delphi/Client/uStart.pas
unit uStart;

interface

uses
  Windows, Messages, SysUtils, Classes, Graphics, Controls, Forms,
  Dialogs, StdCtrls, ExtCtrls;

type
  TfmStart = class(TForm)
    Panel1: TPanel;
    bnOK: TButton;
    bnCancel: TButton;
    rgAction: TRadioGroup;
    procedure FormShow(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  fmStart: TfmStart;

implementation


{$R *.dfm}


procedure TfmStart.FormShow(Sender: TObject);
begin
 ModalResult := mrOK;
 bnOK.SetFocus;
end;

end.
